mcollina/fastify-gql
# Add ability to filter data based on user query

Implementation #56 
 
## Testing

1. Run `node ./examples/queryFilters.js`   
2. Use query in playground                                                  
```

 fragment Doggs on Dog{
   name   
   mother{
       name 
   }
 }

query { 
  dogs(limit:4) {
     ...Doggs
   }
}
```
3. We should see the following output
```
SQL ROOT QUERY: select "name","type" from Dog
SQL RELATION QUERY: select "name" from DoggyParents
```

# Tasks

- [x] Basic implementation
- [x] Experiment with format (relationships)
